src/app/services/context_retrieval.py

```
@@ -0,0 +1,333 @@
+            ContextVerse(
+                row.bible_text_id,
+                row.book_code,
+                row.chapter_number,
+                row.verse_number,
+                row.source_text,
+                row.target_text,
+            )
+            for row in combined_rows
+        ]
+
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)